### ISSUE TITLE
Sort accounts in aws configure sso

### ISSUE THREAD

**Issue by krzysztof-magosa** [aws/aws-cli#6108]
**Is your feature request related to a problem? Please describe.**
At the moment `aws configure sso` displays select box with all accounts in random order.
This is problematic having hundreds of accounts.

**Describe alternatives you've considered**
Search functionality would be even better, but sorting still makes sense.

**Describe the solution you'd like**
Alphabetic ordering would be great.

**Comment by kdaily** [comment]
Thanks for the feature request, @krzysztof-magosa.

**Comment by ryansonshine** [comment]
@kdaily , is the aws team open to a PR for this?

**Comment by ryansonshine** [comment]
@krzysztof-magosa , I went ahead and made the assumption that we would want case-insensitive sorting on PR #6205 . If you had other ideas though I'm open to hearing them.

**Comment by kdaily** [comment]
Thanks @ryansonshine, We'll take a look and review.

**Comment by ruicovelo** [comment]
Yes, sorting helps... But I have thousands of accounts to go through. So for me, search is the way to go. I can't use it any other way.

**Comment by LozanoMatheus** [comment]
Can we add the sort in the roles as well? I also thought about having the `--account-id` as a parameter in the CLI, should it be in a new feature request or ...?

**Comment by YAMLcase** [comment]
What's a possible workaround for this?  I need to set up sso for an account and have thousands of unsorted accounts to scroll through.  Can we at least get a clue how the items are ordered?

**Comment by RyanFitzSimmonsAK** [comment]
This is now supported starting in version `2.34.24` of the AWS CLI.  Please feel free to reopen this issue if you have any problems using this feature.  Thank you for contributing!
